pandas/tests/indexes/period/test_period_range.py

```diff
@@ -113,34 +119,6 @@ def test_construction_from_string(self, freq_offset, freq_period):
         result = period_range(start=end, end=start, freq=freq_period, name="foo")
         tm.assert_index_equal(result, expected)
 
-    def test_construction_from_string_monthly(self):
```

Comment:
Thank you for noticing code duplication in `pandas/tests/indexes/period/test_period_range.py`. I am not sure how it’s related to this bug.  Could you please move these changes in separate PR?